### PR TITLE
Samples: add accessibility description for pushpins, minor refactors

### PR DESCRIPTION
**Pull request by rkoten**
### Description of the changes:
- Android: set pushpin content description, minor refactors
- iOS: set pushpin accessibility label where no title is present

### How changes were validated:
Tested on a Samsung Galaxy S8+ (Android 9), on an iPhone 11 (iOS 13).

### CODE REVIEW

File: Android/samples/sdksample/app/src/main/java/com/microsoft/maps/sample/sdksample/MainActivity.java [hunk 1]
```diff
@@ -68,13 +67,11 @@ protected void onCreate(Bundle savedInstanceState) {
...
 
-        // Enable translucent status bar if supported by Android version.
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
-            getWindow().addFlags(WindowManager.LayoutParams.FLAG_TRANSLUCENT_STATUS);
-            getWindow().getDecorView().setSystemUiVisibility(
-                    View.SYSTEM_UI_FLAG_LAYOUT_STABLE |
-                    View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN);
-        }
+        // Enable translucent status bar.
+        getWindow().addFlags(WindowManager.LayoutParams.FLAG_TRANSLUCENT_STATUS);
```
Comment (JuanPapas) [inline Android/samples/sdksample/app/src/main/java/com/microsoft/maps/sample/sdksample/MainActivity.java:71]: Note that this is being deprecated in Android 11
https://developer.android.com/reference/android/view/WindowManager.LayoutParams#FLAG_TRANSLUCENT_STATUS

File: Android/samples/sdksample/app/src/main/java/com/microsoft/maps/sample/sdksample/MainActivity.java [hunk 2]
```diff
@@ -211,6 +208,9 @@ private void addPin(Geopoint location, String title) {
         pushpin.setTitle(title);
         pushpin.setImage(mPinImage);
         pushpin.setNormalizedAnchorPoint(new PointF(0.5f, 1f));
+        if (title.isEmpty()) {
+            pushpin.setContentDescription("Pushpin");
```
Comment (JuanPapas) [inline Android/samples/sdksample/app/src/main/java/com/microsoft/maps/sample/sdksample/MainActivity.java:212]: Perhaps add the lat, long coordinate? E.g.  "Pushpin at x y"

File: iOS/samples/sdksample/BingMapsSdkSample/ViewController.swift [hunk 3]
```diff
@@ -101,6 +101,7 @@ class ViewController: UIViewController, UIPickerViewDelegate, UIPickerViewDataSo
                     pushpin.image = self.pinImage
                     pushpin.normalizedAnchorPoint = CGPoint(x: 0.5, y: 1.0)
                 }
+                pushpin.accessibilityLabel = "Pushpin"
```
Comment (JuanPapas) [inline iOS/samples/sdksample/BingMapsSdkSample/ViewController.swift:104]: Same suggestion for iOS.  

Also, is there a corresponding change for search?

File: Android/samples/sdksample/app/src/main/java/com/microsoft/maps/sample/sdksample/MainActivity.java (hunk 2 above)
Comment (rkoten) [reply Android/samples/sdksample/app/src/main/java/com/microsoft/maps/sample/sdksample/MainActivity.java]: Agreed offline to enumerate untitled pins instead.

File: iOS/samples/sdksample/BingMapsSdkSample/ViewController.swift (hunk 3 above)
Comment (rkoten) [reply iOS/samples/sdksample/BingMapsSdkSample/ViewController.swift]: Included search.

**Pull request merged**